eCourse_backend/Readme.md

```diff
@@ -1,22 +1,38 @@
 # File requirements.txt:
+
 ```
 pip install -r requirements.txt
 ```
 * is needed for automated django test
 * please add all requirements that are needed for the Django tests to this file, otherwise they won't run!!!
 
 # Running the App
+
 Before running the app and after installing the dependencies, doing the necessary database migrations is mandatory.
 ```
 python manage.py migrate
 ```
 
-# Remarks regarding Folders
-* **/eCourse_backend/static/:**
-    * Contains:
-        * static files
-            * Subfolders for CSS and JS
-        * different Versions of the Logo + some other files
-            * used to create compability with different browser features (some are just for Android/Windows/...)
-            * other files are also needed for these features (e.g. site.webmanifest)
-            * created with: [RealFaviconGenerator](https://realfavicongenerator.net/)
\ No newline at end of file
+# Running tests
+
+```
+./manage.py test
+```
+
+## Coverage
+
+Install with pip
```

Comment:
can it be included in the requirements.txt? Or is there a need to run pip install coverage separately?

Reply:
Coverage is not needed to run the app and therefore not a real `requirement`. But I can include it